### DISCUSSION TITLE
HS.Local anything to take into account?

### DISCUSSION THREAD

**Discussion by Kosock** [intersystems/git-source-control#266] [Q&A]
We have classes in HS.Local which is maped and used in diffrent namespaces.

Is there anything we should consider while using git-source-control?
Like is it necessary that changes to these classes should only be done in the HSCUSTOM namespace?

**Comment by isc-tleavitt** [comment]
The thing to avoid is having multiple repos in different namespaces with different versions of the things in HSCUSTOM.

Do you currently have multiple namespaces with their own repos?

**Reply by Kosock** [reply]
No we did not set it up for the other namespaces yet and thats why I asked the question (since we noticed without git-source-control installed on the other namespace there where synchronization problems what the source-control "thought" needs to be checked in if the code was changed in another namespace where HS.Local was linked. 

So your recommendation would be to use the same repo for all the namespaces? Does the git-source-control differentiate namespace specific stuff in order not to mix it up when using the same repo?

**Reply by isc-tleavitt** [reply]
The standard approach at this point would be to have a repo per namespace, and a separate repo for HSCUSTOM. We have it on the backlog to support a single repo used instance-wide.
